pkg/kubectl/resource_printer.go

```diff
@@ -1480,6 +1480,28 @@ func (h *HumanReadablePrinter) PrintObj(obj runtime.Object, output io.Writer) er
 	w := tabwriter.NewWriter(output, tabwriterMinWidth, tabwriterWidth, tabwriterPadding, tabwriterPadChar, tabwriterFlags)
 	defer w.Flush()
 	t := reflect.TypeOf(obj)
+	// if obj is an api.List object, we first extract its underlying items and then print them individually
+	if t.String() == "*api.List" {
```

Comment:
I'm confused why PrintObj is getting passed a List.  It should never do so if the Visitor pattern is being used.